model2/0A1_GRCTRL_LRAT_ORAT_BASE_MODEL2_MSW.DATA

```diff
@@ -0,0 +1,300 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2018 Equinor
+
+-- This case is a modefied version of model2. This case has standard wells, 3 producers and 1 injector
+
+-- Group PROD har LRAT control, and is limited by ORAT
```

Comment:
Mix of English and Norwegian? Please update the description so that it is self-explanatory what this case tests.

Reply:
Thanks for the modification, but it still does not describe the case, and the spelling mistake is still there. I suggest something like this:

-- This case is a modification of the base model 2 case adding group control
-- Wells are multisegment
-- Group PROD is under LRAT control and is limited by ORAT